spec/actions_manager_spec.coffee

```diff
@@ -92,10 +92,21 @@ action_reporting_tests = ->
       expect(@payload.actions[0]).to.be.an('object')
 
     it 'has proper data in the action\'s object', ->
+      if reported_position != undefined
+        if stringified
+          data = JSON.parse(@data)
+          data.rpos = reported_position
+          data = JSON.stringify(data)
+        else
+          data = JSON.parse(JSON.stringify(@data)) # clone
+          data.rpos = reported_position
```

Comment:
I'm getting a weird fail. I have sent `VALID_LINE_ITEM_DATA_1` with [stringified=false](https://github.com/skroutz/analytics.js/pull/75/commits/b364fdd259667c92c47660cf9f4bcc51c9003756#diff-c993b20c956420a37460c74f28e981df6b2a282e94d45be4da57a97b63dd687bR537) and [rpos=0](https://github.com/skroutz/analytics.js/pull/75/commits/b364fdd259667c92c47660cf9f4bcc51c9003756#diff-c993b20c956420a37460c74f28e981df6b2a282e94d45be4da57a97b63dd687bR577), but the spec fails. Error is `expected { Object (category, type, ...) } to deeply equal { Object (category, type, ...) }`, but If we stringify `data` the spec runs fine. I think it has to do with how object comparison works in CoffeeScript/JavaScript.

```
data.rpos = reported_position
data = JSON.stringify(data)
```

Reply:
you are right, [`_builBeaconPayload`](https://github.com/skroutz/analytics.js/blob/master/src/actions_manager.coffee#L74) always use `JSON.stringify`!

Reply:
Actually stringified param seems redundant.
It is not used anywhere so i will ✂️